Duplicati/Library/Main/Database/Database schema/13. Performance Indexes.sql

```diff
@@ -0,0 +1,4 @@
+﻿CREATE INDEX "FileLookupMetadataID" ON "FileLookup" ("MetadataID");
```

Comment:
I think there is value in adding sql comments indicating the operations this is known to improve so others working on this later can see the reason for the addition.